Aramit2023.csl

```diff
@@ -0,0 +1,252 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="de">
+  <info>
+    <title>Arachnologische Mitteilungen</title>
+    <title-short>AraMit</title-short>
+    <id>http://csl.mendeley.com/styles/styles/Aramit2023</id>
```

Comment:
change to: `<link href="http://www.zotero.org/styles/arachnologische-mitteilungen" rel="self"/>`

Reply:
How can I edit the submitted code?

Reply:
Found it!